### PR TITLE
Add function for parsing port ranges

### PR DESCRIPTION
**Pull request by arjunrn**
Add a function for parsing a comma separated list of port ranges.

### CODE REVIEW

**Comment by mikkeloscar** [comment]
:+1:

File: provisioner/template.go [hunk 1]
```diff
@@ -308,3 +309,29 @@ func accountID(account string) (string, error) {
...
+		}
+		fromPort, err := strconv.Atoi(splitR[0])
+		if err != nil {
+			return nil, fmt.Errorf("invalid start port: %s in input: %s", splitR[0], ranges)
+		}
+		toPort, err := strconv.Atoi(splitR[1])
+		if err != nil {
+			return nil, fmt.Errorf("invalid end port: %s in input: %s", splitR[1], ranges)
+		}
+		p[i] = PortRange{FromPort: int(fromPort), ToPort: int(toPort)}
```
Comment (szuecs) [inline provisioner/template.go]: maybe check that the chosen values are valid in port ranges?
0-65535

Comment (arjunrn) [reply provisioner/template.go]: I've added validation of port ranges

**Comment by arjunrn** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**